### ISSUE TITLE
when install with treesit-auto, error

### ISSUE THREAD

**Issue by getong** [sogaiu/tree-sitter-janet-simple#3]
error msg:
```
⛔ Warning (treesit): The installed language grammar for janet cannot be located 
or has problems (symbol-error): dlsym(0x45205490, tree_sitter_janet): symbol not found
```
 I built on macos
also see renzmann/treesit-auto#116

**Comment by sogaiu** [comment]
I don't know how treesit-auto works but is it possible [the value for `:lang`](https://github.com/renzmann/treesit-auto/blob/016bd286a1ba4628f833a626f8b9d497882ecdf3/treesit-auto.el#L193) is mistaken?

The grammar that's hosted at this repository is not named `janet` but rather [`janet_simple`](https://github.com/sogaiu/tree-sitter-janet-simple/blob/12bfab7db8a5f5b1d774ef84b5831acd34936071/grammar.js#L52).

I don't think this is an issue with this repository.  I suspect it is a problem of `treesit-auto`.  May be it makes sense to report there?

**Comment by getong** [comment]
ok, i fix it in renzmann/treesit-auto#118

**Comment by sogaiu** [comment]
I'm going to close this issue then :)
